lib/membrane/srtp/decryptor.ex

```diff
@@ -0,0 +1,53 @@
+defmodule Membrane.SRTP.Decryptor do
+  @moduledoc """
+  Converts SRTP packets to plain RTP.
+
+  Requires adding [srtp](https://github.com/membraneframework/elixir_srtp) dependency to work.
+  """
+  use Membrane.Filter
+
+  alias Membrane.{Buffer, RTP}
+
+  def_input_pad :input, caps: :any, demand_unit: :buffers
```

Comment:
Shouldn't it have caps SRTP?

Reply:
nope

Reply:
I'm pretty sure it shouldn't have any -> any caps :man_shrugging:

Reply:
I'm sure input caps need to be `any`, as that's what is received from UDP source. Also, neither RTP nor SRTP caps make sense for output imho. Maybe it's time to create `%ByteStream{content: :unknown | caps}` caps